plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/fileio/ForwardingFileIo.java

```diff
@@ -42,16 +42,23 @@ public class ForwardingFileIo
 
     private final TrinoFileSystem fileSystem;
     private final Map<String, String> properties;
+    private final boolean useFileSizeFromMetadata;
```

Comment:
The field name is a bit confusing - its meaning only becomes clear after reading the caller's code. That said, I'm fine with the current name as long as it stays private.